Source/Helpers/MessageSender.swift

```diff
@@ -15,22 +16,51 @@ public class MessageSender {
     private let queue = DispatchQueue(label: "MessageSender")
 
     private func send(message: Message, additionalData: Data?, to channel: Storage.Channel, date: Date) throws {
+        let card = try channel.getCard()
+
+        let ratchetChannel: RatchetChannel?
+        if channel.type == .singleRatchet {
+            ratchetChannel = try Virgil.ethree.getRatchetChannel(with: card)
+        } else {
+            ratchetChannel = nil
+        }
+
         let exported = try message.exportAsJsonData()
 
-        let card = try channel.getCard()
-        let ciphertext = try Virgil.ethree.authEncrypt(data: exported, for: card)
+        let ciphertext = try self.encrypt(data: exported, for: card, withRacthetChannel: ratchetChannel)
```

Comment:
we can pass here channel type to remove constructing RatchetChannel above and encapsulate Virgil work on encryption in one func. Also, we can consider moving all wrapped encrypt decrypt methods to Virgil class, since it's a wrapper itself